xcell/cls/cl.py

```diff
@@ -274,15 +323,58 @@ def compute_tracer_ccl(self, tr, mapper):
                 bz = A*((1.+z)/(1.+z0))**eta*0.0139/0.013872474
                 ia_bias = (z, bz)
             # Get tracer
-            return ccl.WeakLensingTracer(self.cosmo, dndz=(z, pz),
-                                         ia_bias=ia_bias)
+            ccl_tr = ccl.WeakLensingTracer(self.cosmo, dndz=(z, pz),
+                                           ia_bias=ia_bias)
         elif dtype == 'cmb_convergence':
             # TODO: correct z_source
-            return ccl.CMBLensingTracer(self.cosmo, z_source=1100)
+            ccl_tr = ccl.CMBLensingTracer(self.cosmo, z_source=1100)
         else:
             raise ValueError('Type of tracer not recognized. It can be \
                              galaxy_density, galaxy_shear or \
                              cmb_convergence!')
+        return {'name': tr,
+                'ccl_tr': ccl_tr,
+                'ccl_pr': ccl_pr,
+                'ccl_pr_2pt': ccl_pr_2pt,
+                'with_hm': with_hm,
+                'normed': normed_profile}
+
+    def _get_ccl_pk(self, ccl_tr1, ccl_tr2):
+        if ccl_tr1['with_hm'] or ccl_tr2['with_hm']:
+            if ccl_tr1['name'] == ccl_tr2['name']:
+                pr2pt = ccl_tr1['ccl_pr_2pt']
+            else:
+                pr2pt = None
+            k_s = np.geomspace(1E-4, 1E2, 512)
```

Comment:
Why geometric?

Reply:
logarithmic sampling is a good way of capturing the features of the power spectrum

Reply:
Oooh! I thought geometric was something else. I just saw the numpy [documentation](https://numpy.org/doc/stable/reference/generated/numpy.geomspace.html) and this is the same thing as logarithmic: 

```
Return numbers spaced evenly on a log scale (a geometric progression).

This is similar to logspace, but with endpoints specified directly. Each output sample is a constant multiple of the previous.
```